lib/fastlane/actions/slather.rb

```diff
@@ -128,7 +136,12 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :ignore,
                                        env_name: "FL_SLATHER_IGNORE",
                                        description: "Tell slather to ignore files matching a path",
-                                       optional: true)
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
```

Comment:
We could try, but it made it impossible to write a test case in the spec for it (as I didn't know how to make it detect a file in the tests). It feels like the use of bundler needs to be integrated at a higher level as it is spread out among three different actions. The bundle_installer is happy with ./gemfile or ./Gemfile, and Cocoapods does automate the use of bundle if ./Gemfile is found (the lower case ./gemfile is not accounted for).

I also didn't want to change the default of not using bundle to avoid breaking current builds.

(Keep in mind I know little about ruby).